user-interface/src/case-detail/CaseDetailScreen.tsx

```diff
@@ -446,15 +456,19 @@ export default function CaseDetailScreen(props: CaseDetailProps) {
                         onPillSelection={handleSelectedFacet}
                         onUpdateSelection={handleFacetClear}
                         label="Filter by Summary"
+                        ariaDescription="Select multiple options. Results will update when the dropdown is closed."
+                        ariaLive="off"
```

Comment:
for consistency purposes, we should probably make the `ariaDescription & ariaLive` here `aria-description & aria-live` like we have on the DateRangPicker